addons/escoria-ui-9verbs/game.tscn

```diff
@@ -19,6 +19,15 @@ script = ExtResource( 5 )
 main_menu = NodePath("ui/main_menu")
 pause_menu = NodePath("ui/pause_menu")
 
+[node name="dialog_layer" type="CanvasLayer" parent="."]
+
```

Comment:
You changed the layer here. Originally it was layer 3. Was that intended?

Reply:
Yes it was intended : the bug was that dialog lines were displayed above the pause menu. The dialog_layer value doesn't care about the layer position in the scene tree: bigger values are above, so I lowered it.